### PR TITLE
axios get call for style working, static data rendering

### PR DESCRIPTION
**Pull request by yellowstrings**

### CODE REVIEW

File: client/components/Overview/Product-information.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React, { useState, useEffect } from 'react';
+
+const ProductInformation = (props) => {
+  return (
+    <div>
+      {console.log(props.reviews)}
```
Comment (clwilshire) [inline client/components/Overview/Product-information.jsx:6]: does this console.log work?

File: client/components/Related-items-and-comparisons/Stars.jsx [hunk 2]
```diff
@@ -17,9 +17,9 @@ const Stars = (props) => {
   // For each req.body.results.rating, add value to total, and count each rating
   let totalCount = 0;
   let totalOfRatings = 0;
-  for (i = 0; i < props.reviews.results.length; i++) {
-    totalRatingsCount++;
-    totalOfAllRatings += props.reviews.results[i]
+  for (let i = 0; i < props.reviews.results.length; i++) {
```
Comment (clwilshire) [inline client/components/Related-items-and-comparisons/Stars.jsx:20]: Confirm with Terri this isn't a conflict

File: webpack.config.js [hunk 3]
```diff
@@ -9,6 +9,10 @@ module.exports = {
     path: path.resolve("dist"),
     publicPath: "/",
   },
+  devServer: {
```
Comment (clwilshire) [inline webpack.config.js:12]: when do we use devServer?

**Review by clwilshire — COMMENTED** [review]
Looks good to me, need to confirm two Related-iitems-and-comparisons are functional

**Review by TeresaGobble — APPROVED** [review]

**Pull request merged**